ChangeLog.md

```diff
@@ -1,10 +1,150 @@
 ##2016.08.03 version 2.0.0
+* Removal of Force parameters
```

Comment:
As First bullet:  This release contains breaking change.  Please see the migration guide here (link to the gist) for details.